wesnoth-ei.po

```diff
@@ -771,33 +755,28 @@ msgstr ""
 #. Dacyn doesn't really express much and doesn't like to explain himself
 #: data/campaigns/Eastern_Invasion/scenarios/01_Eastern_Invasion.cfg:699
 msgid "Gweddry? Ah, you survive."
-msgstr ""
+msgstr "Gweddry? Ach, jsi naživu."
 
 #. [message]: speaker=Gweddry
 #: data/campaigns/Eastern_Invasion/scenarios/01_Eastern_Invasion.cfg:703
 msgid ""
 "Dacyn, why did you flee the battle? Our situation has become rather "
 "desperate."
-msgstr ""
+msgstr "Dacyne, proč jsi utekl z bitvy? Naše situace byla poměrně zoufalá."
 
 #. [message]: speaker=Dacyn
 #: data/campaigns/Eastern_Invasion/scenarios/01_Eastern_Invasion.cfg:707
-#, fuzzy
-#| msgid ""
-#| "There is no time to explain. If you wish to survive, you will escape "
-#| "through this trapdoor."
 msgid ""
 "Now is not the time for a lengthy report. Unless you intend to die, I "
 "suggest you escape through this trapdoor."
 msgstr ""
-"Není čas na vysvětlování. Jestli chceš přežít, uteč skrz tyhle padací dveře."
+"Není čas na dlouhé popisování. Pokud nemáte v úmyslu zemřít, navrhuji vám "
```

Comment:
Přijde mi to vhodnější, ale přežiju, když to neproskočí.
```suggestion
"Není čas na dlouhé hlášení. Pokud nemáte v úmyslu zemřít, navrhuji vám "
```

Reply:
Dejme tomu.